test/integration/targets/cnos_user/tests/common/sanity.yaml

```diff
@@ -0,0 +1,112 @@
+---
+- debug: msg="START connection={{ ansible_connection }} cnos_user parameter test"
+
+
+- block:
+  - name: Create user
+    cnos_user: &configure
+      name: netend
+      configured_password: Hello!234
+      update_password: on_create
+      roles: network-operator
+#      provider: "{{ connection }}"
+      state: present
+    register: result
+
+  - assert: &true
+      that:
+        - 'result.changed == true'
+
+  - block:
+    - name: conf idempotency
+      cnos_user: *configure
+      register: result
+
+    - assert: &false
+        that:
+          - 'result.changed == false'
+#    when: idem
+
+  - name: Remove user
+    cnos_user: &remove
+      name: netend
+#      provider: "{{ connection }}"
+      state: absent
+    register: result
+
+  - assert: *true
+
+  - name: remove idempotency
+    cnos_user: *remove
+    register: result
+
+  - assert: *false
+
+  - debug: msg="skipping sshkey test as the key needs to be created on the server first"
+
+#  - name: create a new user
```

Comment:
Do you need to have these commented test cases in the test.

Reply:
I have removed them. Thanks